modules/standard/GPU.chpl

```diff
@@ -25,26 +25,30 @@
     This module is unstable and its interface is subject to change in the
     future.
 
+    GPU support is a relatively new feature to Chapel and is under active
+    development.
+
+    For the most up-to-date information about GPU support see the
+    :ref:`technical note <readme-gpu>` about it.
 */
 module GPU
 {
+  pragma "no doc"
   extern proc chpl_gpu_write(const str : c_string) : void;
 
   /*
-     This function is intended to be called from within a GPU
-     kernel and is useful for debugging purposes.
+     This function is intended to be called from within a GPU kernel and is
+     useful for debugging purposes.
 
-     Currently writeln() will make a loop ineligable for
-     gpuization. This function is here as a stopgap measure
-     until we fix that.
+     Currently using :proc:`ChapelIO.write` to send output to ``stdout`` will
+     make a loop ineligible for GPU execution; use :proc:`gpuWrite` instead.
 
-     Currently this function will only work if values of type
-     c_string are passed.
+     Currently this function will only work if values of type c_string are
```

Comment:
Let's surround `c_string` in double backticks so that it displays like code in the documentation